TrkHitReco/src/CombineStrawHits_module.cc

```diff
@@ -98,6 +102,8 @@ namespace mu2e {
     _maxR2(     config().maxR()*config().maxR()),
     _minT(      config().minT()),
     _maxT(      config().maxT()),
+    _overrideminTOff(      config().minTOff(_minTOff)),
```

Comment:
Because mintoff is optional, I  believe _minTOff will be uninitialized in case it isn't set. It shouldn't be used, but it might be cleaner to initialize it to _minT in that case.